src/Routers/UsersRouter.js

```diff
@@ -25,7 +26,20 @@ export class UsersRouter extends ClassesRouter {
     let data = deepcopy(req.body);
     req.body = data;
     req.params.className = '_User';
-    return super.handleCreate(req);
+
+    if (req.config.verifyUserEmails) {
```

Comment:
Actually I guess we don't have a UsersController yet. We could add one later and clean this up at that time.